iceoryx2-cxx/include/iox2/entry_value.hpp

```diff
@@ -112,7 +112,7 @@ inline auto update(EntryValue<S, KeyType, ValueType>&& self) -> EntryHandleMut<S
     iox2_entry_value_update(self.take_handle_ownership(), nullptr, &entry_handle_mut_handle);
 
     EntryHandleMut<S, KeyType, ValueType> entry_handle_mut(entry_handle_mut_handle);
-    return std::move(entry_handle_mut);
+    return entry_handle_mut;
```

Comment:
@elBoberido I removed std::move due to copy elision prevention and I was wondering why this was not showing before? ` [-Wpessimizing-move]`

Reply:
Is the warning active? AFAIK it is not explicitepy turned on but might  be part of `Wall`